front/src/common/BootstrapSNCF/OptionsSNCF.tsx

```diff
@@ -47,7 +48,11 @@ export default function OptionsSNCF({
               value={option.value}
               checked={selectedValue === option.value}
             />
-            <label className="options-btn font-weight-medium" htmlFor={`${name}${option.value}`}>
+            <label
+              className="options-btn font-weight-medium"
+              htmlFor={`${name}${option.value}`}
+              data-testid={`${name}${option.value}-label`}
```

Comment:
What about adding a `-` to improve lisibility ?
```suggestion
              data-testid={`${name}-${option.value}-label`}
```